### PR TITLE
ptBR docs translate and new property of buttongroup buttons[] array

### PR DESCRIPTION
**Pull request by lukBarros**

### CODE REVIEW

**Comment by samdark** [comment]
Will merge it when there will be no English files in the commit...

**Comment by lukBarros** [comment]
Ok!

**Comment by lukBarros** [comment]
@samdark 100% in ptBR. Marge it!

File: ButtonGroup.php [hunk 1]
```diff
@@ -43,6 +43,7 @@ class ButtonGroup extends Widget
      * which can be specified as a string or an array of the following structure:
      *
      * - label: string, required, the button label.
+	 * - visible: boolean, optional, whether this button item is visible. Defaults to true.
```
Comment (samdark) [inline ButtonGroup.php]: It should not be in this pull request.

Comment (lukBarros) [inline ButtonGroup.php]: How to undo?

Comment (samdark) [inline ButtonGroup.php]: Nevermind, I'll merge it.

Comment (lukBarros) [inline ButtonGroup.php]: Ok das próximas vezes serei mais cuidadoso, me perdoe  venho de pelo menos 8 anos de svn.

**Comment by samdark** [comment]
Merged https://github.com/yiisoft/yii2-bootstrap/commit/b8c381f8d9ccaf541de061e1858bb0144639fd14. Thanks!

**Pull request closed without merging**